recipe/meta.yaml

```diff
@@ -12,14 +12,16 @@ source:
     - conda-build.patch
 
 build:
-  number: 7
+  number: 0
```

Comment:
hmm, is there already build # 7 present? if so, you will need to set number to 8, instead or zero

Reply:
![image](https://user-images.githubusercontent.com/373868/207894375-a215e573-22e4-4ba6-b118-56b798d51cbf.png)
We do not currently have msmpi on defaults, I thought it would be fair to reset the build number.